client/src/components/RestaurantShow.js

```diff
@@ -2,16 +2,21 @@ import React, { useState, useEffect } from "react";
 import { Link } from "react-router-dom"
 
 import ReviewList from "./ReviewList.js"
+import NewReviewForm from "./NewReviewForm.js"
+import ErrorList from "./layout/ErrorList.js";
 
 const RestaurantShow = (props) => {
+
+  const { currentUser } = props
```

Comment:
you should destructure this on line 8

`const RestaurantShow = ({ currentUser }) => {`